File: elixir/clockhands.exs
Enum.each 0..10, fn i ->
  t = div(43200 * i + 21600, 11)
  [h, m, s] = [div(t, 3600), rem(div(t, 60), 60), rem(t, 60)]
  IO.puts to_string(
    :io_lib.format("~2..0b:~2..0b:~2..0b",
    [if h == 0 do 12 else h end, m, s]))
end

File: julia/clockhands.jl
using Printf

seconds(i) = div(43200 * i + 21600, 11)
zeroTo12(h) = h == 0 ? 12 : h
parts(t) = (zeroTo12(div(t, 3600)), rem(div(t, 60), 60), rem(t, 60))
0:10 .|> seconds .|> parts .|> t -> @printf("%02d:%02d:%02d\n", t...)